app/controllers/posts_controller.rb

```diff
@@ -1,16 +1,16 @@
 class PostsController < ApplicationController
   def index
-    @user = current_user
-    @posts = current_user.last_three_posts
+    @user = User.includes(:posts).find(params[:user_id])
+    @page = params.fetch(:page, 0).to_i
+    @posts = @user.posts.offset(@page * POSTS_PER_PAGE).limit(POSTS_PER_PAGE)
```

Comment:
- [ ] You already solve the n+1 problem in the show method. Also, do the same thing @posts variable.  Example: `@user.posts.includes(:comments)`
- [ ] Please fix the error when see all posts. 
 ![image](https://user-images.githubusercontent.com/38635158/167236463-0e62f227-dc95-49c7-a6c1-9ea9e3a83600.png)
- [ ] Also see all posts button is not working. You should fix it too.